components/locid_transform/src/provider.rs

```diff
@@ -61,6 +62,26 @@ pub struct StrStrPair<'a>(
     #[cfg_attr(feature = "serde", serde(borrow))] pub Cow<'a, str>,
 );
 
+#[zerovec::make_varule(StrStrStrPairVarULE)]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Deserialize),
+    zerovec::derive(Deserialize)
+)]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, databake::Bake),
+    zerovec::derive(Serialize),
+    databake(path = icu_locid_transform::provider),
+)]
+/// A triplet of strings with a EncodeAsVarULE implementation.
+pub struct StrStrStrPair<'a>(
+    pub Language,
```

Comment:
Because `StrStrStrPairVarULE` doesn't have the same binary representation as `StrStrPairVarULE`, this isn't going to be compatible across Postcard binaries. It will need a new `AliasesV2`.